packages/web/components/Setup/SetupPersonalityType.tsx

```diff
@@ -60,63 +63,130 @@ export const SetupPersonalityType: React.FC<SetupPersonalityTypeProps> = ({
     }
 
     onNextPress();
+  }, [colorMask, onNextPress, toast, updateAboutYou, user]);
+
+  // mask should always only have at most a single bit set
+  const toggleMaskElement = (mask = 0): void => {
+    setColorMask((current = 0) => {
+      if ((mask & current) > 0) {
+        // if the bit in mask is set
+        return current & ~mask; // unset it
+      }
+      return current | mask; // otherwise set it
+    });
   };
 
   return (
-    <FlexContainer>
-      <MetaHeading mb={5} textAlign="center">
-        Personality Type
-      </MetaHeading>
-      <Text mb={10}>
-        {`Please select your personality type below. Not sure what type you are? `}
-        <MetaLink href="https://enneagramtest.net/" isExternal>
-          Take a quick test.
-        </MetaLink>
-      </Text>
-      <SimpleGrid columns={[1, null, 2, 3]} spacing="8">
-        {personalityTypeChoices.map((p: PersonalityType) => (
-          <HStack
-            key={p.id}
-            p={6}
-            spacing={4}
-            bgColor={
-              personalityType && personalityType.id === p.id
-                ? 'purpleBoxDark'
-                : 'purpleBoxLight'
-            }
-            borderRadius="0.5rem"
-            _hover={{ bgColor: 'purpleBoxDark' }}
-            transition="background 0.25s"
-            cursor="pointer"
-            onClick={() => setPersonalityType(p)}
-            border="2px"
-            borderColor={
-              personalityType && personalityType.id === p.id
-                ? 'purple.400'
-                : 'transparent'
-            }
+    <FlexContainer maxW="100%">
+      <Flex direction="column">
+        <MetaHeading mb={5} textAlign="center">
+          Person&#xAD;ality Type
+        </MetaHeading>
+        <Text mb={10}>
+          Please select your personality components below. Not sure what type
+          you are?
+          <Text as="span"> Take </Text>
+          <MetaLink
+            href="//dysbulic.github.io/5-color-radar/#/explore/"
+            isExternal
           >
```

Comment:
This quiz is sweet; nice work

Reply:
¿The first one with the sliders made sense? I thought it shouldn't take much explanation, but you never know with a UI how intuitive others will find it.